packages/next-on-pages/package.json

```diff
@@ -53,7 +53,7 @@
 		"cookie": "^0.5.0",
 		"esbuild": "^0.15.3",
 		"js-yaml": "^4.1.0",
-		"miniflare": "^3.20231002.1",
+		"miniflare": "^3.20231030.2",
```

Comment:
why does next-on-pages depend on miniflare??? 😮

Reply:
ah ok it is because the next-dev internal package is actually added as a sub-module of the next-on-pages package 😅